Include/internal/pycore_stackref.h

```diff
@@ -63,11 +63,11 @@ extern void _Py_stackref_associate(PyInterpreterState *interp, PyObject *obj, _P
 
 static const _PyStackRef PyStackRef_NULL = { .index = 0 };
 
-#define PyStackRef_None ((_PyStackRef){ .index = 1 } )
```

Comment:
Can you add a comment here what 1 and 3 are reserved for (presumably the bitmasks).